### PR TITLE
Update passwordless RFDs with current FIDO2 flows

### PR DESCRIPTION
**Pull request by codingllama**
Update FIDO2 authn flows for `tsh` and clear the implication that authenticators are only capable of returning a single assertion.

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

**Comment by codingllama** [comment]
Friendly ping @alistanis @r0mant ?

File: rfd/0040-webauthn-support.md [hunk 1]
```diff
@@ -40,35 +42,35 @@ to the server with the signed challenge. Assuming all is well this completes the
 registration process.
 
 ```
-                          Registration flow
```
Comment (codingllama) [inline rfd/0040-webauthn-support.md:43]: Note to reviewers: everything below this point is just automatic whitespace trimming.

**Comment by codingllama** [comment]
Friendly ping @alistanis @r0mant ?

**Review by r0mant — APPROVED** [review]

**Review by alistanis — APPROVED** [review]
Sorry for the delay, I was on vacation until yesterday and I'm getting caught up on reviews

**Pull request merged**